packages/components/psammead-radio-schedule/src/index.jsx

```diff
@@ -107,19 +113,20 @@ through the list to render a star-time and program-card, inside a gird.
 We intend to move the map functionality out of psammead in a future iteration.
 */
 const RadioSchedule = ({ schedules, dir, ...props }) => (
-  <Grid dir={dir} {...schedulesGridProps}>
+  <StyledGrid forwardedAs="ul" dir={dir} {...schedulesGridProps}>
```

Comment:
Should this be using normal div when there is just one radio schedule?

Reply:
I just spoke to Tom IRL, this is not a case we are currently considering.

Reply:
Do you mean that there will always be more than one item? Just saying because the rest of our components follow that approach cc @greenc05

Reply:
> Do you mean that there will always be more than one item?

yep, we expect there to always be more than one item.